superset-frontend/src/explore/components/controls/MetricsControl.jsx

```diff
@@ -337,7 +337,11 @@ export default class MetricsControl extends React.PureComponent {
         <OnPasteSelect
           isMulti={this.props.multi}
           name={`select-${this.props.name}`}
-          placeholder={t('choose a column or aggregate function')}
+          placeholder={
+            this.props.multi
+              ? t('choose one or more column or aggregate function')
+              : t('choose a column or aggregate function')
```

Comment:
@junlincc to confirm, you're going to revisit this text at a later date, correct?